integration-cli/docker_cli_pull_local_test.go

```diff
@@ -421,3 +421,26 @@ func (s *DockerRegistryAuthHtpasswdSuite) TestPullWithExternalAuth(c *check.C) {
 
 	dockerCmd(c, "--config", tmp, "pull", repoName)
 }
+
+// TestRunImplicitPullWithNoTagOnlyPullDefaultTag should pull implicitely only the default tag (latest)
+func (s *DockerRegistrySuite) TestRunImplicitPullWithNoTagOnlyPullDefaultTag(c *check.C) {
```

Comment:
I know you are only moving the test, but can you shorten the name to `TestRunImplicitPullWithNoTag`?